code/modules/awaymissions/zlevel.dm

```diff
@@ -58,7 +58,7 @@ var/global/list/potentialRandomZlevels = generateMapList(filename = "config/away
 			late_setup_level(block(locate(1, 1, zlev), locate(world.maxx, world.maxy, zlev)))
 			space_manager.remove_dirt(zlev)
```

Comment:
Yes. It doesn't work. And https://github.com/ParadiseSS13/Paradise/blob/c973875b95ed4a58415cc3e341d0b8952f6a933b/code/modules/space_management/zlevel_manager.dm#L77 does not seem to reference either hooks, or powernets. This PR makes it work.